docs/dev/apis/csync/publishing.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Since this file/article is so short, I recommend combining this into the usage-guide file